pkg/discovery/probe/pod_probe_util.go

```diff
@@ -56,6 +56,27 @@ func GetResourceRequest(pod *api.Pod) (cpuRequest float64, memRequest float64, e
 	return
 }
 
+// Get the UUID that will be used in Turbonomic server. Here we build the UUID based on pod UID, namespace and name.
+// The out UUID is in format "UID:namespace:name"
+func GetTurboPodUUID(pod *api.Pod) string {
```

Comment:
":" is not allowed in UID, namespace or name.
If I define the followng namespace spec
```json
{
  "kind": "Namespace",
  "apiVersion": "v1",
  "metadata": {
    "name": "develo:pment",
    "labels": {
      "name": "development"
    }
  }
}
```
it will return me:
```console
$ kc create -f examples/yaml/namespace.json
The Namespace "develo:pment" is invalid:
* metadata.name: Invalid value: "develo:pment": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
* metadata.labels: Invalid value: "develo:pment": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```